src/vstart.sh

```diff
@@ -160,11 +160,7 @@ overwrite_conf=0
 cephx=1 #turn cephx on by default
 gssapi_authx=0
 cache=""
-if [ `uname` = FreeBSD ]; then
-    objectstore="filestore"
-else
-    objectstore="bluestore"
-fi
+objectstore="bluestore"
```

Comment:
I recall FreeBDS doesn't support bluestore, maybe adding message in that case?

Reply:
It means if using FreeBDS, error occurred, right?

Reply:
@wjwithagen Can you chime in?  With filestore dropped in Reef, is there an ObjectStore backend can be used on FreeBSD?  If bluestore just won't work, perhaps we could default to memstore here for FreeBSD?

Reply:
I don't have environment to test FreeBSD, but I have modified it to memstore, is it ok?

Reply:
I don't have such an environment either.  It would be good to hear from @wjwithagen, or perhaps @trociny knows?

Reply:
OK, wait for response, thank you so much.

Reply:
I have not tried Ceph on FreeBSD for long time, and I am sure @wjwithagen would know better, but AFAIR the bluestore is not supported on FreeBSD due to libaio dependency. I doubt it has changed recently, which means that Reef (server) will not be available for FreeBSD. 

I have no idea if memstore is supported on FreeBSD.

Taking that this vstart issue would be a highly minor problem for FreeBSD comparing with others I would not do anything FreeBSD specific here and leave it for people who want to work on making FreeBSD work.

Reply:
> Taking that this vstart issue would be a highly minor problem for FreeBSD comparing with others I would not do anything FreeBSD specific here and leave it for people who want to work on making FreeBSD work.

Well, we already have a FreeBSD-specific branch in `vstart.sh` and need to do something about it...  If @wjwithagen doesn't chime in the course of a few days, I'd support a not-tested switch to memstore.

Reply:
Alternatively, we could simply remove the `if [ `uname` = FreeBSD ]; then` statement...

Reply:
Okay, I agree with you.

Reply:
Hi @idryomov , is it time to merge this first? I prefer keeping FreeBSD's objectstore as memstore, if it goes wrong, someone could raise a PR then.